backend/ethereum/channel/funder_test.go

```diff
@@ -46,6 +46,44 @@ const (
 	txGasLimit = 100000
 )
 
+func TestFunder_RegisterAsset_IsAssetRegistered(t *testing.T) {
+	n := 2
+
+	rng := pkgtest.Prng(t)
+	simBackend := test.NewSimulatedBackend()
+	ksWallet := wallettest.RandomWallet().(*keystore.Wallet)
+	cb := ethchannel.NewContractBackend(simBackend, keystore.NewTransactor(*ksWallet, types.NewEIP155Signer(big.NewInt(1337))))
+	funder := ethchannel.NewFunder(cb)
+	assets := make([]ethchannel.Asset, n)
+	depositors := make([]ethchannel.Depositor, n)
+	accs := make([]accounts.Account, n)
+	for i := 0; i < n; i++ {
+		assets[i] = *(wallettest.NewRandomAddress(rng).(*ethwallet.Address))
+		accs[i] = accounts.Account{Address: ethwallet.AsEthAddr(wallettest.NewRandomAddress(rng))}
+	}
```

Comment:
Can you move the setup code into a separate function so that the code of this method focuses on the actual test cases?

Reply:
Updated in this [force-push](https://github.com/hyperledger-labs/go-perun/compare/c5255fbe5f5453796953eb1c9e559a9565a7c2ea..4a12ab6d8062a3296b14d92a697dc705aec0b7e5).